src/lib/services/validator/lcd.ts

```diff
@@ -34,3 +47,67 @@ export const getValidatorDataLcd = (
       `${endpoint}/cosmos/staking/v1beta1/validators/${encodeURI(validatorAddr)}`
     )
     .then(({ data }) => parseWithError(zValidatorResponseLcd, data).validator);
+
+export const getValidatorStakingProvisionsLcd = async (
+  endpoint: string,
+  chain: string
+) => {
+  if (chain === "osmosis") {
+    const [mintParams, epochProvisions] = await Promise.all([
+      getMintParamsLcd(endpoint, chain),
+      getEpochProvisionsLcd(endpoint, chain),
+    ]);
+
+    const stakingProportion =
+      mintParams.params.distributionProportions?.staking;
+
+    if (!stakingProportion) throw new Error("Staking proportion not found");
+
+    const epochStakingProvisions =
+      epochProvisions.epochProvisions.mul(stakingProportion);
+
+    const epochIndentifier = mintParams.params.epochIdentifier;
+
+    if (epochIndentifier === "hour")
+      return { stakingProvisions: epochStakingProvisions.mul(8760) };
+
+    if (epochIndentifier === "day")
+      return { stakingProvisions: epochStakingProvisions.mul(365) };
+
+    if (epochIndentifier === "week")
+      return { stakingProvisions: epochStakingProvisions.mul(52) };
+
+    return { stakingProvisions: big(0) };
+  }
+
+  const [distParams, annualProvisions] = await Promise.all([
+    getDistributionParamsLcd(endpoint),
+    getAnnualProvisionsLcd(endpoint, chain),
+  ]);
+
+  const stakingProvisions = annualProvisions.annualProvisions.mul(
+    big(1).sub(distParams.params.communityTax)
+  );
+
+  return { stakingProvisions };
+};
+
+export const getValidatorDelegatorsLcd = async (
+  endpoint: string,
+  validatorAddress: ValidatorAddr,
+  isInitia: boolean
+): Promise<ValidatorDelegatorsResponse> => {
+  const route = isInitia
+    ? "initia/mstaking/v1/validators"
+    : "cosmos/staking/v1beta1/validators";
+
+  return axios
+    .get(
+      `${endpoint}/${route}/${encodeURI(validatorAddress)}/delegations?pagination.limit=1&pagination.count_total=true`
```

Comment:
Use this way instead of the above one
```ts
axois.get(endpint, {params: {
          "pagination.limit": 1,
          "pagination.count_total": true
        }})
```